content/TrueCommand/Alerts/AlertManage.md

```diff
@@ -88,24 +96,22 @@ To create a new alert rule, click **+ NEW ALERT RULE** and follow the creation w
 
 To create a new rule:
 
-1. Specify the **Alert Options**:
+1. Specify the **Alert Options**.
    a. Enter a name into the **Alert Rule Name** field.
    b. Select a system from the **System** drop-down.
      The rule applies to the selected system(s). Non-administrative user accounts require appropriate system permissions.
    c. Select the alert type on the **Priority** drop-down list. Choose **Information**, **Warning**, or **Critical** to determine the alert category generated.
    d. Type a description for the alert.
-2. Specify the **Alert Triggers**:
-   Select a data source or rule type from the drop-down list to determine what can trigger an alert. For example, *cpu_temperature* means the alert rule monitors the temperature of the chosen system. Scroll down the list to find the desired source.
+2. Specify the **Alert Triggers**.
+   Select a data source or rule type from the dropdown list to determine what can trigger an alert. For example, *cpu_temperature* means the alert rule monitors the temperature of the chosen system. Scroll down the list to find the desired source.
 
    ![AlertTriggerDataSources](/images/TrueCommand/2.1/AlertTriggerDataSources.png "Alert Trigger Data Sources")
 
-   b. Select the comparison type from the **Comparator** drop-down list (**Greater Than**, **Less Than**, or **Not Equals**). The comparison type applies to the data source and comparison value.
+   b. Select the comparison type from the **Comparator** dropdown list (**Greater Than**, **Less Than**, or **Not Equals**). The comparison type applies to the data source and comparison value.
    c. Specify the comparison value by entering an integer appropriate for the selected options in the **Value** field. The integer acts as a threshold or limitation for when the rule generates an alert.
```

Comment:
Line 111 item "c." not breaking to new line.

Reply:
Fixed "a" that was not showing and "c" in a new commit.